atlasdb-config/src/test/java/com/palantir/atlasdb/factory/TransactionManagersTest.java

```diff
@@ -315,7 +309,8 @@ public void batchesRequestsIfBatchingEnabled() throws InterruptedException {
 
     @Test
     public void doesNotBatchRequestsIfBatchingNotEnabled() {
-        setUpTimeLockBlockInInstallConfig();
+        when(config.namespace()).thenReturn(Optional.of(CLIENT));
```

Comment:
consider putting this `when` in setup rather than repeating it in several tests.